### PR TITLE
[Feature] Support clipping min_lr in StepLrUpdaterHook to help MMDet3D

### LINKED ISSUE #994: [Feature] Support min_lr clipping in lr_scheduler (MMDet3D related problem)
**Hi, I am Ziyi, a developer of MMDet3D. Can we support learning rate clipping in lr_scheduler? For example, when I use StepLR which decays by 0.7 every 10 epochs with a `init_lr==1e-3`, I would like to keep my `lr==1e-5` after it decays to some values lower than `1e-5`. Such min_lr clipping is common in several 3D point cloud papers.**

**Motivation**
For example, in the PointNet++ official code, they decay lr by 0.7 every 10 epochs and clip a minium lr of 1e-5 as shown [here](https://github.com/charlesq34/pointnet2/blob/master/scannet/train.py#L85). This is widely adopted in other point cloud related papers like PointNet (see [here](https://github.com/charlesq34/pointnet/blob/master/train.py#L80)), DGCNN, etc.

**Related resources**
I think it's quite easy to implement. We can add a `min_lr` args to `LrUpdaterHook`, and use a `max(lr, self.min_lr)` to determine the return value in `get_lr`? However, I am not sure whether it will cause other questions because I am not very familiar with MMCV.

**Additional context**
Is there any existing way to achieve my feature request? If you not yet and you think my solution is okay, I can create a PR to support it. Thanks!

### PR DESCRIPTION
**Pull request by Wuziyi616**
Fix #994. Please refer to that issue for detailed motivation. This feature will be used in MMDet3D. Thanks!

### CODE REVIEW

**Review by zhouzaida — APPROVED** [review]
LGTM. If passing CI, it could be merged.

File: mmcv/runner/hooks/lr_updater.py [hunk 1]
```diff
@@ -165,7 +165,7 @@ def get_lr(self, runner, base_lr):
 @HOOKS.register_module()
 class StepLrUpdaterHook(LrUpdaterHook):
 
-    def __init__(self, step, gamma=0.1, **kwargs):
+    def __init__(self, step, gamma=0.1, min_lr=None, **kwargs):
```
Comment (ZwwWayne) [inline mmcv/runner/hooks/lr_updater.py:168]: Unit tests should be added to verify this feature.

Comment (ZwwWayne) [inline mmcv/runner/hooks/lr_updater.py:168]: Suggest adding docstring for this module accordingly.

Comment (Wuziyi616) [reply mmcv/runner/hooks/lr_updater.py:179]: Done.

Comment (Wuziyi616) [reply mmcv/runner/hooks/lr_updater.py:179]: Yes.

**Comment by Wuziyi616** [comment]
Seems that I can't run the workflow? May need @ZwwWayne to approve it.

**Comment by ZwwWayne** [comment]
> Seems that I can't run the workflow? May need @ZwwWayne to approve it.

lint failed.

**Comment by Wuziyi616** [comment]
The linting check error said that "mmcv/engine/test.py:35:80: E501 line too long (113 > 79 characters)", which is absolutely wrong because I didn't even modify that file. I think this PR is ready for merge @ZwwWayne?

**Review by ZwwWayne — APPROVED** [review]
